source/documentation/compatibility.rst

```diff
@@ -27,38 +31,116 @@ The following table summarises the available features in Ubermag, OOMMF an
      - ``micromagneticmodel``
      - ``oommfc``
      - ``mumax3c``
-   * - Exchange energy
+   * -
+     - Multiple energy terms of the same type
```

Comment:
We should probably say how to do it in Ubermag. Here is one possible suggestion.

```suggestion
   * - Multiple energy terms of the same type
     - User has to specify unique `name`s
```